coldfront/core/project/tests/test_views/test_auto_approval_removal.py

```diff
@@ -155,13 +156,21 @@ def test_project_join_request_email(self):
                              role__name__in=['Manager', 'Principal Investigator'],
                              status__name='Active')]
 
-        email_body = f'User {self.user1.first_name} {self.user1.last_name} ' \
-                     f'({self.user1.email}) has requested to join your ' \
-                     f'project, {self.project1.name} via MyBRC user portal. ' \
-                     f'Please approve/deny this request.'
+        domain = import_from_settings('CENTER_BASE_URL')
```

Comment:
@jofeinstein FYI: I should've caught this earlier, but this test needed to be (and is now) updated.